docs/source/tutorials/keras-text-classifiers.rst

```diff
@@ -0,0 +1,10 @@
+.. _keras-text-classifiers-tutorial:
+
+.. note::
+
+    This tutorial is intended to be run in an IPython notebook.
+    It is also available as a notebook file here_.
+
+.. _here: https://github.com/TeamHG-Memex/eli5/blob/master/notebooks/keras-text-classifiers.ipynb
```

Comment:
Wondering if this link should be updated to point to the new org: `https://github.com/eli5-org/eli5/blob/master/notebooks/keras-text-classifiers.ipynb`. I know that other links will still be pointing to the old repo, but if this PR is only getting merged here then the link will be broken

Reply:
Aha good point - I wanted to avoid updating the links unless it's needed (as I hope we can get back to old repo), but in this case it's indeed better to update, thanks for the heads up 👍